src/admin.rs

```diff
@@ -56,6 +57,24 @@ pub async fn list_checkpoints(
     Ok(manifest.core.checkpoints)
 }
 
+/// Deletes all objects with the specified prefix. This includes all
+/// "subdirectories" objects, since object stores are not hierarchical.
+pub async fn delete_objects_with_prefix(
+    object_store: Arc<dyn ObjectStore>,
+    prefix: &Path,
+) -> Result<(), Box<dyn Error>> {
+    let stream = object_store
+        .list(Some(prefix))
+        .map_ok(|m| m.location)
+        .boxed();
+    object_store
+        .delete_stream(stream)
```

Comment:
this is a little scary if you accidentally pass in the wrong path 😬 I know this is how most of these tools work, but do we want to have some kind of dummy file we create that indicates the dir was created by the bencher and then check for the presence of that file before running the delete?

Reply:
Good idea. WDYT about creating and checking the dummy file in main.rs and leaving this file as-is so it's more generic?

Reply:
👍 yeah definitely wouldn't make sense in the admin, I like your suggestion